CODEOWNERS

```
@@ -1 +1,3 @@
 *       @aalavian @anantsimran @chrisochoatri @isaak-willett @marccarre
+
+CODEOWNERS @aalavian @marccare
```

Typo: `marccare` -> `marccarre`.
```suggestion
CODEOWNERS @aalavian @marccarre
```